### DISCUSSION TITLE
Displaying Current Context in the menubar instead of app icon

### DISCUSSION THREAD

**Discussion by benhughes** [ttscoff/bunch-docs#99] [Ideas]
Have you thought about displaying an icon/emoji/letter in the menu bar when your context is active? I like the idea of glancing up and seeing what you're supposed to be focused on.

<img width="188" alt="Screenshot 2021-07-05 at 13 04 49" src="https://user-images.githubusercontent.com/1478194/124468882-96818880-dd91-11eb-987b-62b4cff436bd.png">

It could be a frontmatter attribute. example:
display when active: 🤙

then when the bunch is active it replaces the bunch icon with the selected emoji.

**Comment by ttscoff** [comment]
Except you can have multiple bunches open at any given time…

- Brett

**Comment by benhughes** [comment]
Was thinking it would only work in Single Bunch mode or be optional using the latest bunch.
